lib/modules/userInfo.js

```diff
@@ -209,6 +212,27 @@ async function showUserInfo(card) {
 		userHTML += string.escape`<div class="${isHighlight ? 'blueButton' : 'redButton'}" id="highlightUser" data-userid="${data.id}">${isHighlight ? i18n('userInfoHighlight') : i18n('userInfoUnhighlight')}</div>`;
 	}
 
+	if (data.name) {
+		userHTML += string.escape`<a target="_blank" rel="noopener noreferer" id="blockUser" class="blueButton">${i18n('userInfoBlock')}</a>`;
+
+		$body.on('click', '#blockUser', () => {
+			loggedInUserHash().then(
```

Comment:
If the `loggedInUserHash` doesn't a return hash (like when you're not logged in), the button will useless. It might be better to only add the button if `hash` is truish.
```
const hash = await loggedInUserHash();
if (hash) { /* ... add button */ }
```